build.sbt

```diff
@@ -6,15 +6,16 @@ lazy val slickjdbcextension = (project in file(".")).
     name := "slick-jdbc-extension",
     organization := "com.github.tarao",
     version := "0.0.7",
-    scalaVersion := "2.11.8",
+    scalaVersion := "2.11.11",
+    crossScalaVersions := Seq("2.11.11", "2.12.3"),
 
     // Depenency
     libraryDependencies ++= Seq(
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-      "com.typesafe.slick" %% "slick" % "3.1.1",
-      "com.github.tarao" %% "nonempty" % "0.0.6",
-      "org.scalatest" %% "scalatest" % "2.2.6" % "test",
-      "org.scalamock" %% "scalamock-scalatest-support" % "3.2.2" % "test",
+      "com.typesafe.slick" %% "slick" % "3.2.1",
+      "com.github.tarao" %% "nonempty" % "0.0.7",
+      "org.scalatest" %% "scalatest" % "3.0.5" % "test",
+      "org.scalamock" %% "scalamock" % "4.0.0" % "test",
```

Comment:
`scalamock-scalatest-support` seems not be maintained (and not support scala 2.12) and we can use ScalaMock 4.0.0 without `scalamock-scalatest-support`

http://scalamock.org/user-guide/installation/

> Before version 4.0, ScalaMock had different artefacts for ScalaTest and Specs2 (scalamock-scalatest-support and scalamock-specs2-support), which actually pulled in versions of ScalaTest and Spec2 respectively. Since ScalaMock version 4.0.0, this was changed to combine both these adaptors into the main jar file and make the transitive dependencies optional.